client/overview/modal/connection-success/index.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
```

Comment:
please remove this as it's not used anymore

Reply:
Removed!